scripts/FullRunRPS.json

```diff
@@ -0,0 +1,56 @@
+{
+    "Remote": {
+        "Exe": "quicperf",
+        "Arguments": "-selfsign:1"
+    },
+    "FullMatrix": false,
+    "Tests": [
+        {
+            "TestName": "RPS",
+            "Local": {
+                "Platform": "Windows",
+                "Tls": ["stub", "schannel", "openssl", "mitls"],
+                "Arch": ["x64", "x86", "arm", "arm64"],
+                "Exe": "quicperf",
+                "Arguments": "-test:RPS -target:$RemoteAddress"
+            },
+            "Variables": [
```

Comment:
Eventually I'll probably want to add more than just these, but this is a good start. We can see how long it takes to run and go from there.